src/login/components/Buttons/ButtonModal.js

```diff
@@ -0,0 +1,27 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+import Button from "reactstrap/lib/Button";
+
+class ButtonModal extends Component {
```

Comment:
I'm not sure about this component. The only thing that differenciates it from just using straight up `reactstrap/lib/Button` is making `className` and `onClick` required - but omitting any of them would result in very obvious bugs, so I'd think we don't need type checking to prevent them.
Perhaps it's useful to have it if it's just a stub that will be further developed.
In any case, it would be more idiomatic to put `<Button {...this.props}>`

Reply:
This is one of the new concept we will be introducing to the codebase where we are breaking down elements into smaller reusable components:
- By packaging this button into its own component as a `ButtonModal` we will be able to clearly differentiate it from, for instance, a primary button (that holds the primary call to action of a page) i.e. we will no longer have to look at the dispatching action to differentiate between buttons. 
- The namig comes from having a folder with multiple kinds of Buttons of which this is the one for modals i.e. `ButtonPrimary`, `ButtonSecondary` and `ButtonModal` 

The required props are also a part of a new more helpful way of building components because I'd rather want a warning about the onClick missing in the console than having to search through the code myself. Having said that, the final required proptypes are not complete yet.